updated meetup discussions to stay current

nquinlan opened the pull request:
at present the README links to the initial (?) February meetup, this updates it to make sure it's evergreen.

nquinlan committed:
updated meetup discussions to stay current

at present the README links to the initial (?) February meetup, this updates it to make sure it's evergreen.

terinjokes commented:
The link to the open issues should include "&state=open". GitHub will show closed issues if that was the last view you looked at.

nquinlan committed:
updated current issues link to remain open

as per @terinjokes' comment in #5, without `state=open` Github will default to the user's previous state (open or closed), this ensures it will always show only open issues.

nquinlan commented:
nice catch, thanks
